.github/workflows/pr_verification.yml

```diff
@@ -15,6 +15,35 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+
+  dump_contexts_to_log:
```

Comment:
I assume this is just for debugging and will be removed before merge?

Reply:
Yes. I forgot to remove it. Done.